README.md

```diff
@@ -1,7 +1,13 @@
 # Next-gen Slack platform project template - Reverse String
-This repo contains a sample project and embedded lightweight SDK of a Typescript based project for the new Deno runtime. 
 
-The main file that brings it all together is the `project.ts` file.  So far `functions`, `workflows`, `triggers` and `tables` are supported and those should each be created in a file per, under each corresponding directory. `functions/reverse.ts` has a simple sample. After you create a new function or workflow make sure you add it to the `Project` object in `project.ts`. 
+This repo contains a sample project and embedded lightweight SDK of a Typescript
+based project for the new Deno runtime.
+
+The main file that brings it all together is the `project.ts` file. So far
+`functions`, `workflows`, `triggers` and `tables` are supported and those should
```

Comment:
Is the workflows and triggers mention here still accurate?

Reply:
Heh, no. Neither is tables. Nor do we have a `project.ts` file anymore. These changes are just auto-formatting caught by running `deno fmt`.

My next task is to do a docs sweep on the repos, so that'll handle updating these!

Reply:
Sg!